README.md

```diff
@@ -0,0 +1,36 @@
+# Install OMERO on CentOS 7 or Ubuntu 18.04 using Ansible
```

Comment:
The README will ultimately be the same for any of these repositories, right?

Reply:
Yes, though instead of duplicating it I think it would be better to have it in once place and link to it from the other repos.

Reply:
Makes sense. Do you have a suggestion on the place?

Reply:
Keep it here? This is the simplest example and therefore the recommended entry point.

Reply:
K. Guess let's start there and we can see where we go. Another option might be an eventual overview sphinx/readthedocs ansible page.

Reply:
Since we do not have decided the final location of those instructions, I think we should hold off on including it  in docs.o.org.
I will check the instructions in the coming days

Reply:
OK. Do you have any more suggestions for this PR?

Reply:
As indicated I need to go over the instructions